infra-as-code/bicep/modules/policy/definitions/lib/china/policy_definitions/_mc_policyDefinitionsBicepInput.txt

```diff
@@ -1,388 +1,392 @@
 {
-  name: 'Append-AppService-httpsonly'
-  libDefinition: json(loadTextContent('lib/china/policy_definitions/policy_definition_es_mc_append_appservice_httpsonly.json'))
+	name: 'Append-AppService-httpsonly'

```

Comment:
Seems like ecessive indent changes? could you fix to make easier to review?

Reply:
need to fix the script instead, this file is auto-generated